gel-tokio/examples/transaction_errors.rs

```diff
@@ -18,7 +18,7 @@ fn check_val0(val: i64) -> anyhow::Result<()> {
 
 fn check_val1(val: i64) -> Result<(), CounterError> {
     if val % 3 == 1 {
-        if thread_rng().gen_bool(0.1) {
```

Comment:
Why this change?

Reply:
I bumped rand as part of the change (we use it in gel-auth) as well. It's a bit annoying because they have changed the API pretty heavily between versions. The old API was deprecated.